config/apidoc.php

```diff
@@ -14,6 +14,20 @@
      */
     'router' => 'laravel',
 
+    /*
+     * URL prefix of routes for your documentation.
+     *
+     * By default, `/doc` opens the blade, and `/doc.json` returns the Postman collection.
+     */
+
+    'url_prefix' => '/doc',
```

Comment:
I have a problem with these two being top-level keys. They only apply to the ``laravel` documentation type, so it's not fair to have them at the top level.

Reply:
I combined the route settings in one block.
Since these settings relate to routes, I preferred to transfer them to the `routes` block ([see](https://github.com/andrey-helldar/laravel-apidoc-generator/blob/patch-2019-12-17-11-42/config/apidoc.php#L57-L79)).